cclient/request/get_transactions_to_approve.c

```diff
@@ -10,6 +10,7 @@
 get_transactions_to_approve_req_t* get_transactions_to_approve_req_new() {
   get_transactions_to_approve_req_t* req =
       (get_transactions_to_approve_req_t*)calloc(1, sizeof(get_transactions_to_approve_req_t));
+  req->reference = (flex_trit_t*)calloc(1, sizeof(flex_trit_t) * FLEX_TRIT_SIZE_243);
```

Comment:
`reference` is optional, that's why it's not allocated in `get_transactions_to_approve_req_new`.
It is allocated only if needed in `get_transactions_to_approve_req_set_reference`.